src/main.jl

```diff
@@ -10,8 +10,15 @@ function sim!(P, C, dt)
 end
 
 function sim!(P, C; dt = 0.1ms, duration = 10ms)
-    for t = 0ms:dt:(duration - dt)
-        sim!(P, C, dt)
+    sized = duration/dt
```

Comment:
The CI failure is because `sized` is one shorter than the actual number of integrations.

```suggestion
    sized = length(0ms:dt:duration)
```